ParisBNF/abb/BNFabb3.xml

```diff
@@ -0,0 +1,327 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BNFabb3" type="mss">
+  <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Psalter</title>
+            <editor key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed
+            under the Creative Commons Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+            <date>2016-11-02T12:04:37.552+02:00</date>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno>BnF Éthiopien d'Abbadie 3</idno>
+                  </msIdentifier>
+
+               <msContents>
+                  <summary/>
+
+                  <msItem xml:id="ms_i1">
+                     <locus from="1r"/>
+                     <title type="complete" ref="LIT5852Psalter" cert="low">Psalter</title>
```

Comment:
I would be interested to know why you think that this is a non-standard Psalter. Because ʾAnqaṣa bərhān is missing?

Reply:
Exactly. But it described in a very general way. I understand that Conti Rossini did not consider it as smth special. However, he does not comment on the absense of Anqasa berhan.  I think, he might have overlooked AB, or the part with AB is missing, but he did not comment.

Reply:
Do you think it should be recorded as normal Psalter, but incomplete? It will be equally uncertain.

Reply:
I can, though, not problem about this.

Reply:
I have no definitive answer of course, maybe this would be a case for a `<note>` with a statement that makes this reasoning clear for the readers, whatever ID you decide for.

Reply:
I did.